edx_event_bus_kafka/config.py

```diff
@@ -0,0 +1,59 @@
+"""
+Configuration loading and validation.
+"""
+
+from typing import Optional
+
+import warnings
+import attr
+from confluent_kafka.schema_registry import SchemaRegistryClient
+from django.conf import settings
+
+
+@attr.s
+class CommonConfig:
+    settings = attr.ib(type=dict)
+    
+
+def create_schema_registry_client() -> Optional[SchemaRegistryClient]:
+    """
+    Create a schema registry client from common settings.
+    """
+    url = getattr(settings, 'EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL', None)
+    if url is None:
+        warnings.warn("Cannot configure event-bus-kafka: Missing setting EVENT_BUS_KAFKA_SCHEMA_REGISTRY_URL")
+        return None
+
+    key    = getattr(settings, 'EVENT_BUS_KAFKA_SCHEMA_REGISTRY_API_KEY', '')
+    secret = getattr(settings, 'EVENT_BUS_KAFKA_SCHEMA_REGISTRY_API_SECRET', '')
+
+    return SchemaRegistryClient({
+        'url': url,
+        'basic.auth.user.info': f"{key}:{secret}",
+    })
+
+def load_common_settings(settings) -> Optional[dict]:
+    """
+    Load common settings, a base for either producer or consumer configuration.
+    """
+    bootstrap_servers = getattr(settings, 'EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS', None)
+    if bootstrap_servers is None:
+        warnings.warn("Cannot configure event-bus-kafka: Missing setting EVENT_BUS_KAFKA_BOOTSTRAP_SERVERS")
+        return None
+
+    settings = {
```

Comment:
This will need a new name so that it doesn't conflict with the Django settings, and so that the next few lines are pulling from Django settings.

Reply:
gotcha, do we want bootstrap_servers to call it from settings and then key/secret from the other settings?

Reply:
with smth like bootstrap_settings as the name